AK/Time.h

```diff
@@ -454,6 +459,67 @@ struct UnixDateTime : Detail::NaiveTime {
     }
 };
 
+// Monotonic time represents time returned from the CLOCK_MONOTONIC clock, which has an arbitrary fixed reference point.
+struct MonotonicTime : private Detail::NaiveTime {
+    // Monotonic time does not have a defined reference point.
+    // A MonotonicTime at the reference point is therefore meaningless.
+    MonotonicTime() = delete;
+    constexpr MonotonicTime(MonotonicTime const&) = default;
+    constexpr MonotonicTime(MonotonicTime&&) = default;
+    constexpr MonotonicTime& operator=(MonotonicTime const&) = default;
+    constexpr MonotonicTime& operator=(MonotonicTime&&) = default;
+
+#ifndef KERNEL
+    [[nodiscard]] static MonotonicTime now();
+    [[nodiscard]] static MonotonicTime now_coarse();
+#endif
+
+    [[nodiscard]] i64 seconds() const
+    {
+        return offset.to_seconds();
+    }
+    [[nodiscard]] i64 milliseconds() const { return offset.to_milliseconds(); }
+    [[nodiscard]] i64 nanoseconds() const { return offset.to_nanoseconds(); }
```

Comment:
Yes; this is extremely strange since even if I manually remove all white space and make the format identical to the below, it still gets wrapped, even though the line is not even as long.